product_cost_security/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Product Cost Security",
     "summary": "Product cost security restriction view",
-    "version": "15.0.1.0.0",
+    "version": "15.0.1.0.1",
```

Comment:
Why manually changing the version? Does this require migration script when coming from v14? If the `id` of the XML-ID has been preserved, you don't need to do anything.